test/integration/runner_common.h

```diff
@@ -10,6 +10,8 @@
 #include <iostream>
 #include <string>
 
+namespace fs = std::filesystem;
```

Comment:
It's generally bad practice to have using declarations or namespace aliases in a header file as they then pollute the namespaces of anything that includes the header.

Things are normally fully-qualified in the header file and then we define and use the namespace aliases in the CPP file instead.